config/initializers/letter_opener.rb

```diff
@@ -0,0 +1,5 @@
+if Object.const_defined? :LetterOpener
```

Comment:
Letter opener doesn't exist in any environment other than `test` so we need to check that it's there before we initialize its config